Makefile

```diff
@@ -7,3 +10,9 @@ run:
 run-grpc:
 	@go run main.go start -g true -r 3939
 
+test-dep:
+	@cd examples/ && go run main.go 2>/dev/null && cd - & sleep 5
+
+test: test-dep
+	@echo "=========RUNNING UNIT TESTS==========="
+	@go test $(TESTABLE_PACKAGES) -coverprofile coverprofile.out -failfast && pkill main
```

Comment:
It is not a good idea to kill using regex in a Makefile. I'd rather retrieve the pid from `test-dep` and use it with `kill <PID>`. If I run pitaya-admin's tests I'd rather not kill all my other main processes.